packages/openapi3/src/types.ts

```diff
@@ -431,7 +431,7 @@ export type OpenAPI3Schema = Extensions & {
    *
    * @see https://tools.ietf.org/html/draft-wright-json-schema-validation-01#section-6.23
    */
-  enum?: string[];
+  enum?: (string | number | boolean)[];
```

Comment:
Why is this changee here?

Reply:
I added a type check in the openapi.ts and this type wasn't accurate